build_runner/lib/src/daemon/daemon_builder.dart

```diff
@@ -98,11 +100,12 @@ class BuildRunnerDaemonBuilder implements DaemonBuilder {
     await _buildOptions.logListener.cancel();
   }
 
-  void _logMessage(Level level, String message) {
-    // TODO(grouma) - use the logic in std_io_logging.dart to include color.
-    _outputStreamController.add(ServerLog((b) => b.log =
-        LogRecord(level, message, 'BuildRunnerBuildDaemon').toString()));
-  }
+  void _logMessage(Level level, String message) => overrideAnsiOutput(
+      true,
```

Comment:
I think we want to retain the `assumeTty == true || ansiOutputEnable` logic if possible

Reply:
I don't think that makes sense here because this message is being sent over a websocket. The daemon client is running in another process which presumably will be printing these messages.

Reply:
It's not outrageously expensive, but each call to `overrideAnsiOutput` creates a `Zone` to run inside. Should we be doing that a higher level than each log message?

Reply:
I was looking at that but couldn't think of a better location. Thoughts?